CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 * Enhanced `markdown` component to support the underlying `components` prop from
   `react-markdown`. Use this prop to customize markdown rendering.
 
+### 🐞 Bug Fixes
+* The `Record.descendants` and `Record.allDescendants` getters were incorrectly returning the
+ record itself. This has been fixed.
```

Comment:
Ah interesting - there is a chance there is some code out there that was depending on the prior behavior (intentionally or not), but hard to argue that it was correct. This looks like the right choice / fix to me.